lib/cisco_node_utils/cmd_ref/bgp_neighbor.yaml

```diff
@@ -114,6 +114,13 @@ password_type:
     get_value: '/^password (\d+)/'
     default_value: 0
 
+peer_type:
+  _exclude: [ios_xr, N3k, N5k, N6k, N7k, N9k-F]
```

Comment:
Sai just added a key for `N3k-F` which needs to be added to your `_exclude`.

Reply:
added